src/wallet/rpc/spend.cpp

```diff
@@ -1159,6 +1160,267 @@ RPCHelpMan send()
+                    UniValue rkvp(UniValue::VOBJ);
+                    rkvp.pushKV(receiver.get_str(), 0);
+                    receiver_key_value_pairs.push_back(rkvp);
+                    addresses_without_amount.insert(receiver.get_str());
+                } else {
+                    receiver_key_value_pairs.push_back(receiver);
+                }
+            }
+
+            if (addresses_without_amount.size() == 0) {
+                throw JSONRPCError(RPC_INVALID_PARAMETER, "Must provide at least one address without a specified amount");
+            }
+
+            CCoinControl coin_control;
+
+            SetFeeEstimateMode(*pwallet, coin_control, options["conf_target"], options["estimate_mode"], options["fee_rate"], /* override_min_fee */ false);
+
+            coin_control.fAllowWatchOnly = ParseIncludeWatchonly(options["include_watching"], *pwallet);
+
+            bool lock_unspents = false;
+            if (options.exists("lock_unspents")) {
+                lock_unspents = options["lock_unspents"].get_bool();
+            }
+
+            bool rbf = pwallet->m_signal_rbf;
+            if (options.exists("replaceable")) {
+                rbf = options["replaceable"].get_bool();
+            }
+
+            FeeCalculation fee_calc_out;
+            CFeeRate fee_rate = GetMinimumFeeRate(*pwallet, coin_control, &fee_calc_out);
+            // Do not, ever, assume that it's fine to change the fee rate if the user has explicitly
+            // provided one
+            if (coin_control.m_feerate && fee_rate > *coin_control.m_feerate) {
+               throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Fee rate (%s) is lower than the minimum fee rate setting (%s)", coin_control.m_feerate->ToString(FeeEstimateMode::SAT_VB), fee_rate.ToString(FeeEstimateMode::SAT_VB)));
+            }
+            if (fee_calc_out.reason == FeeReason::FALLBACK && !pwallet->m_allow_fallback_fee) {
+                // eventually allow a fallback fee
+                throw JSONRPCError(RPC_WALLET_ERROR, "Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable -fallbackfee.");
+            }
+
+            CMutableTransaction rawTx = ConstructTransaction(NullUniValue, receiver_key_value_pairs, options["locktime"], rbf);
+            LOCK(pwallet->cs_wallet); // Lock automatically released at end of function
+            std::vector<COutput> all_the_utxos;
+            AvailableCoins(*pwallet, all_the_utxos, &coin_control, /* sweep 0-value utxos*/ 0);
+
+            const bool sendmax = options.exists("sendmax") && options["sendmax"].get_bool();
+            CAmount total_input_value(0);
+            for (const COutput& output : all_the_utxos) {
+                CHECK_NONFATAL(output.nInputBytes > 0);
+                if (sendmax && fee_rate.GetFee(output.nInputBytes) > output.tx->tx->vout[output.i].nValue) {
+                    continue;
+                }
+                CTxIn input(output.tx->GetHash(), output.i);
```

Comment:
Needs to handle `rbf` and set the right sequence number

```suggestion
                CTxIn input(output.tx->GetHash(), output.i, CScript(), rbf ? MAX_BIP125_RBF_SEQUENCE : CTxIn::SEQUENCE_FINAL);
```

Reply:
Added, thanks.